docs/quickstart_guide.md

```diff
@@ -152,6 +152,7 @@ SSD-equipped nodes are highly recommended. Full list of CloudLab nodes can be fo
 3. Start `firecracker-containerd` in a background terminal named `firecracker`:
     ```bash
     sudo PATH=$PATH screen -dmS firecracker bash -c "/usr/local/bin/firecracker-containerd --config /etc/firecracker-containerd/config.toml > >(tee -a /tmp/vhive-logs/firecracker.stdout) 2> >(tee -a /tmp/vhive-logs/firecracker.stderr >&2)"
+    sudo PATH=$PATH screen -dmS gvisor bash -c "/usr/local/bin/gvisor-containerd --address /run/gvisor-containerd/gvisor-containerd.sock --config /etc/gvisor-containerd/config.toml > >(tee -a /tmp/vhive-logs/gvisor.stdout) 2> >(tee -a /tmp/vhive-logs/gvisor.stderr >&2)"
```

Comment:
I think this will lead to confusion, in my opinion a separate section/ sub-section should be created for gvisor like there is for stock-only. This will be misleading for newcomers as the point above mentions that the script can be used to start firecracker-containerd

Reply:
I agree with Anshal, need a separate note.